bemani/backend/iidx/pendual.py

```diff
@@ -414,6 +421,8 @@ def db_to_game_rank(self, db_dan: int, cltype: int) -> int:
             }[db_dan]
         elif cltype == self.GAME_CLTYPE_DOUBLE:
             return {
+                self.DAN_RANK_7_KYU: self.GAME_DP_DAN_RANK_5_KYU,
```

Comment:
How would this be possible, given that dan ranks don't span multiple games? Also, is it correct to map them all to 5kyu?

Reply:
Well, I ran into this specific issue because the server seems to pull profiles from later games??? That profile from a later game might be 6 or 7 kyu and throw an exception. I figured mapping to 5 kyu was fine but that's based on the assumption the game doesn't support anything below those for DP? Specifically in music_reg, `get_any_profiles` is called which if the user does not have a profile for the given version, will return a profile for the latest version

Reply:
OH, the `get_any_profiles` code, right right. Okay yeah that's a fun edge case. That shouldn't be used much, but I think in some rival cases it comes up. Seems legit to keep a full mapping here then.